### DISCUSSION TITLE
Is it possible to deploy your backend(.NET) to Netlify with github.

### DISCUSSION THREAD

**Discussion by itsdotnet** [community/community#68399] [Programming Help] [answered]
If its not where can I get free server ?

**Comment by melvincwng** [comment]
Hi @itsdotnet, as far as I'm aware, Netlify is a platform to host only frontend or serverless applications. It cannot be used to host backend applications.

**Reply by itsdotnet** [reply, marked as answer]
> Hi @itsdotnet, as far as I'm aware, Netlify is a platform to host only frontend or serverless applications. It cannot be used to host backend applications.

Thanks

**Reply by itsdotnet** [reply]
Thanks. Do you know others Can I deploy
